apps/admin-gui/src/app/facilities/pages/facility-detail-page/facility-detail-page.component.ts

```diff
@@ -46,6 +47,7 @@ export class FacilityDetailPageComponent implements OnInit {
 
       this.facilityManager.getFacilityById(facilityId).subscribe(facility => {
         this.facility = facility;
+        this.entityStorageService.setEntity({id: facility.id, beanName: facility.beanName})
```

Comment:
If you add also facility name to the storage here, you will be able to use it in facility-hosts.component.ts and you won't have to call method `getFacilityById` in facility host component to get facility from backend.

Reply:
As mentioned before this would create possiblity of incorrect data. But we may discuss this further.